app.py

```diff
@@ -872,9 +878,8 @@ def update_yaml():
 
 # *** START SERVER ***
 
-if __name__ == '__main__':
-    # Threaded option to enable multiple instances for multiple user access support
-    if version() == 'DEV':
-        app.run(threaded=True, port=config ['port'], host="0.0.0.0")
-    else:
-        app.run(threaded=True, port=config ['port'])
+# Threaded option enables multiple instances for multiple user access support
+if version() == 'DEV':
```

Comment:
But the check on `version()` here can go. If the `if __name__ == '__main__':` block is taken, we will be in development mode by default and so we don't actually need to check `version()` anymore.

Reply:
Great, just added both your suggestions. Thanks for catching that!